index.html

```diff
@@ -0,0 +1,170 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Home</title>
+    <link rel="stylesheet" href="styles/main.css" >
+  </head>
+  <body>
+    <div class="wrapper">
+      <header>
+        <nav class="nav__social">
+          <a href="#facebook"><img class="bottom__border" src="assets/facebook.png" alt="facebook icon" /></a>
+          <a href="#rss"><img class="bottom__border" src="assets/rss.png" alt="rss icon" /></a>
+          <a href="#twitter"><img class="bottom__border" src="assets/twitter.png" alt="twitter icon" /></a>
+          <a href="#youtube"><img class="bottom__border" src="assets/youtube.png" alt="youtube icon" /></a>
+        </nav>
+
+        <a class="link__subscribe" href="#Subscribe">Subscribe</a>
+
+        <nav class="nav__main">
+          <li class="bottom__border"><a href="#sk8tNews">sk8t news</a></li>
```

Comment:
If you're going to use list items here, you'll need to wrap them in a parent `ul`